scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/task/CommonAttribute.java

```diff
@@ -176,30 +171,10 @@ public void setMaxNumberOfExecution(int numberOfExecution) {
      * These information are transmitted to the policy that can use it for the scheduling.
      *
      * @return generic information.
-     * @deprecated Use {@link #getGenericInformation()} instead.
-     * This method is scheduled for removal in next major release.
-     */
-    @Deprecated
-    public Map<String, String> getGenericInformations() {
```

Comment:
Yes it could break existing workflows,but I supposed it s acceptable it s 9 months this method is deprecated . I will remove it, but I understand it can be risky, so I will highlight the fact with Brian and the all team before the next release is done.